### PR TITLE
Update telemetry about

### PR DESCRIPTION
**Pull request by tpleavitt**
Remove link to the features/telemetry, it no longer exists

### CODE REVIEW

**Comment by tpleavitt** [comment]
@MCamp859 , mainly just removing the link.  Do we want to rewrite the last sentence to move them on to the telemetry guide since we are only going to 1 place now?

No rush on this, just cleaning up lose ends

File: source/clear-linux/concepts/telemetry-about.rst [hunk 1]
```diff
@@ -83,5 +83,3 @@ To put this concept into practice, see the following resources:
 *	:ref:`telem-guide`
 *	`Telemetry feature description`_
```
Comment (MCamp859) [inline source/clear-linux/concepts/telemetry-about.rst]: @tpleavitt  I agree about modifying the last sentence. Suggest something like: 
    To put this concept into practice, refer to :ref:`telem-guide`.

Also delete line 84 (2nd bullet), along with the link definition in lines 86-87.

Comment (tpleavitt) [reply source/clear-linux/concepts/telemetry-about.rst]: Sounds good

Comment (MCamp859) [reply source/clear-linux/concepts/telemetry-about.rst]: @tpleavitt  please check for back-ticks around telem-guide? GH ate them in my response. 
Should be  "":ref:`telem-guide`""

Comment (tpleavitt) [reply source/clear-linux/concepts/telemetry-about.rst]: Oops!  Done

**Comment by tpleavitt** [comment]
@MCamp859 , can you merge or do we need @krisdale  to do the merge?  I can't.

**Comment by krisdale** [comment]
@tpleavitt Looking at it now, will have it merged shortly :)

**Pull request merged**